src/Type/ODMType.php

```diff
@@ -113,7 +113,7 @@ public function convertToDatabaseValue($value, AbstractPlatform $platform): ?str
             $context = $this->getSerializationContext();
             $value   = $this->getSerializer()->serialize($value, $this->format, $context);
         } catch (ExceptionInterface $exception) {
-            $message = sprintf('Serialization exception occurred for class "%s".', $this->getEntityClass());
+            $message = sprintf('Serialization exception occurred for class "%s". (%s)', $this->getEntityClass(), $exception->getMessage());
```

Comment:
Hello! Thanks for PR. I'm suggesting another format.

```suggestion
            $message = sprintf('Serialization exception occurred for class "%s": %s.', $this->getEntityClass(), $exception->getMessage());
```